app/views/qsets/_qset_context.html.erb

```diff
@@ -1,32 +1,28 @@
 <span id="qset_context">
-  <h4>
-    Qset: <%= @qset.name %>
-    <% if can? :update, Qset %>
-      <a class="btn shortcut-button-link"
-         id="edit-qset"
-         data-toggle="modal"
-         data-target="#modal-edit-qset">
-        <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
-      </a>
-      <!-- todo: delete button here? -->
+    <h3 class="center"><b><%= @qset.name %></b>
+    <% if !@qset.parent.nil? %>
```

Comment:
Why can't an admin change the name of an organization (a 'base' qset without a parent) anymore?